### PR TITLE
Add test of ONNX_ATEN

### PR DESCRIPTION
**Pull request by zrphercule**
In #14239 we fixed ONNX_ATEN. 
In order to make sure its correctness in the future, we should add related test case.
We use torch.fmod() to test ONNX_ATEN.

### CODE REVIEW

**Comment by zrphercule** [comment]
Before merging of #14239, we should not merge this one.

**Comment by jamesr66a** [comment]
Can't you just combine the two PRs?

**Comment by zrphercule** [comment]
@jamesr66a yeah you are right... in fact before I finished this pr, that one has already started to land... And due to our sev, it failed to land for many times and now i finished this pr...

File: test/test_jit.py [hunk 1]
```diff
@@ -9109,6 +9109,24 @@ def forward(self, x, y):
             operator_export_type=OperatorExportTypes.ONNX_ATEN_FALLBACK)
         self.assertExpected(exported)
 
+    # torch.fmod is using to test ONNX_ATEN.
+    # If you plan to remove fmod from aten, or found this test failed.
+    # please contact @Rui.
+    @skipIfRocm
+    @skipIfNoLapack
```
Comment (houseroad) [inline test/test_jit.py]: Is this necessary? I mean skipIfNoLapack

**Review by houseroad — APPROVED** [review]
Left some inline comment, otherwise, looks good

**Pull request closed without merging**